### PR TITLE
Add EdgesOnly selection policy

### LINKED ISSUE #10637: Need "Edges" only inspected policy for graphs
Questions on Twitter about "edge" only hover tooltip:

https://twitter.com/DataDrivenAngel/status/1324015994759745536

and AFIAK this is not possible, since there is not an inspection for just `EdgesOnly`

On the assumption that (I think) this is a fairly trivial add, I will tentatively schedule for 2.3

### PR DESCRIPTION
**Pull request by bryevdv**
- [x] issues: fixes #10637
- [x] tests added / passed
- [ ] release document entry (if new feature or API change)

If you have any thoughts about `GraphRendere.get_selection_manager` let me know and I will update the PR. Perhaps it should just raise an error (i.e. things should go through the sub-renderers always instead)

### CODE REVIEW

**Comment by bryevdv** [comment]
@mattpap the eslint failure is now happing in  CI

File: bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts [hunk 1]
```diff
@@ -82,6 +82,79 @@ describe("GraphHitTestPolicy", () => {
     edge_stub.restore()
   })
 
+    describe("EdgesOnly", () => {
```
Comment (mattpap) [inline bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts:85]: Indent.

**Comment by mattpap** [comment]
It fails legitimately, as unit tests have bad indentation.

File: bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts (hunk 1 above)
Comment (bryevdv) [reply bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts:85]: I"d really like to have all the linter/codebase checks in the codebase job

Comment (mattpap) [reply bokehjs/test/unit/models/graphs/graph_hit_test_policy.ts]: They is already `test_eslint` in codebase job and that job failed for the initial pusy (https://github.com/bokeh/bokeh/runs/1356605779?check_suite_focus=true) (along with bokehjs' jobs).

**Pull request merged**